### PR TITLE
Bugfixes: Various bugfixes

### PR DESCRIPTION
**Pull request by Michael-Burke**
- Adjusted jq value for -r instead of -re
- Adjusted image comparison to work with null tags and use `ID` instead of `RepoDigests` which caused issues if multiple digests
- Adjusted usage to `compose_client` over `docker compose` explicit call

### CODE REVIEW

File: src/_clean.sh [hunk 1]
```diff
@@ -66,5 +66,5 @@ function clean_sweepUploadsCache() {
   info "core-backend: Cleaning uploads/exports caches"
   # Leaving the cleanup fairly light, this should help a ton without getting aggressive
   debug "`compose_client exec -T -w /usr/src/plextrac-api/uploads plextracapi \
-    find . -maxdepth 1 -type f -regextype posix-extended -regex '^.*\.(json|xml|ptrac|csv|nessus)$' -delete`"
```
Comment (Michael-Burke) [inline src/_clean.sh:69]: Need to understand better if there was a reason that `-regextype posix-extended` was chosen...

File: src/_clean.sh [hunk 2]
```diff
@@ -66,5 +66,5 @@ function clean_sweepUploadsCache() {
...
+    find . -maxdepth 1 -type f -regex '^.*\.\(json\|xml\|ptrac\|csv\|nessus\)$' -delete`"
```
Comment (Michael-Burke) [inline src/_clean.sh:69]: Gross bash-ism but seems to work

File: src/_clean.sh (hunk 1 above)
Comment (BadgerOps) [reply src/_clean.sh:69]: > Need to understand better if there was a reason that `-regextype posix-extended` was chosen...

https://www.gnu.org/software/findutils/manual/html_node/find_html/posix_002dextended-regular-expression-syntax.html#posix_002dextended-regular-expression-syntax

Comment (Michael-Burke) [reply src/_clean.sh:69]: @BadgerOps Thank you!

File: src/_clean.sh (hunk 2 above)
Comment (Michael-Burke) [reply src/_clean.sh:69]: @BadgerOps does this accomplish the same thing in your opinion?

File: src/_clean.sh (hunk 1 above)
Comment (BadgerOps) [reply src/_clean.sh:69]: No problem. Now, why @jesseops added that is still a mystery that doc strings or a semi snarky GitHub comment could possibly fix.

File: src/_check.sh [hunk 3]
```diff
@@ -108,7 +108,7 @@ function _check_os_supported_flavor_and_release() {
 
 # Check for some base required packages to even validate the system
 function _check_base_required_packages() {
-  requiredCommands=('jq' 'lsb_release' 'wget' 'bc')
+  requiredCommands=('jq' 'lsb_release' 'wget' 'bc' 'docker')
```
Comment (jesseops) [inline src/_check.sh:111]: The utility itself handles docker installation, so ideally shouldn't depend on docker.

Comment (Michael-Burke) [reply src/_check.sh:111]: Totally get the pre-install portion. This isn't complete and we're trying to get it to validate that `docker compose` vs `docker-compose` is working in the environment. That part is incomplete in this PR and a WIP

**Pull request merged**